eng/pipelines/templates/jobs/ci.yml

```diff
@@ -182,7 +182,7 @@ jobs:
         inputs:
           command: custom
           custom: 'tool'
-          arguments: 'install --global --add-source "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" --version "1.0.0-dev.20211119.1" "Azure.Sdk.Tools.SnippetGenerator"'
+          arguments: 'restore'
```

Comment:
Will this work on a system where it isn't previously installed?

Reply:
Yes. It both installs and upgrades, unlike `dotnet tool install` vs. `dotnet tool update`.